File: Aga.ApiPlusAngular/ClientApp/src/app/Models/addDTOs/AddGroupsDTO.ts
export class AddGroupDTo{
    name : string;
    image : string;
    idTeacher: number;

    isValid(): boolean {
        if (
            this.name == "" || this.image == "" || this.idTeacher == NaN
        ) {

            return false
        }
        else {
            return true;
        }
    }
   }

File: Aga.DTO/Add-ting DTOs/AddGroupsDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Aga.DTO.Add_ting_DTOs
{
    public class AddGroupsDTO
    {
        [Required(ErrorMessage = "Name is required")]
        public string Name { get; set; }
        [Required(ErrorMessage = "Image is required")]
        public string Image { get; set; }
        [Required(ErrorMessage = "Id of teacher is required")]
        public string idTeacher { get; set; }
    }
}
